engine/src/org/pentaho/di/job/entries/trans/JobEntryTrans.java

```diff
@@ -877,10 +881,31 @@ public Result execute( Result result, int nr ) throws KettleException {
           }
         }
 
+        SlaveServer remoteSlaveServer = null;
+        TransExecutionConfiguration executionConfiguration = new TransExecutionConfiguration();
+        if ( !Utils.isEmpty( runConfiguration ) ) {
```

Comment:
I looked at the implementations for getJobEntryAttributeString and they appear to all return null when the attribute is not found.